tools/cli/commands.go

```diff
@@ -770,7 +765,7 @@ func SignalWorkflow(c *cli.Context) {
 	})
 
 	if err != nil {
-		ErrorAndExit("Signal workflow failed", err)
+		ErrorAndExit("Signal workflow failed.", err)
 	} else {
 		fmt.Println("Signal workflow succeed.")
```

Comment:
Can you put succeeded instead of succeed for these cases?

Reply:
Updated